src/components/RangeDateField/utils.ts

```diff
@@ -0,0 +1,69 @@
+import type {DateTime} from '@gravity-ui/date-utils';
```

Comment:
its better to create `utils/getRangeEditableSections.ts` file and add to `utils/index.ts`